### ISSUE TITLE
Csi resizer oom killed

### ISSUE THREAD

**Issue by marcinpro1** [openebs/mayastor#1774]
Csi resizer getting oom killer, no matter how much memory i shove into the container.

No valuable information in log.

Maybe linked with this? [https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/1248](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/1248)

Would like to now if i its necessary for this container to work if I won't use disk resize.
Also is there anywhere I should look for more details about this issue in my cluster?

**Comment by tiagolobocastro** [comment]
Would you be able to update the log to trace? This can be done by updating helm vars:
`--set csi.node.logLevel=trace`
And then reproduce again and share a support bundle here? https://openebs.io/docs/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/advanced-operations/supportability

**Comment by marcinpro1** [comment]
Here you go:
[openebs_dump.tar.gz](https://github.com/user-attachments/files/17850632/openebs_dump.tar.gz)
some things were redacted

**Comment by tiagolobocastro** [comment]
> Would like to now if i its necessary for this container to work if I won't use disk resize.

Not sure, I'd say probably not, but since we are exposing resize feature will this cause an issue? You could try removing the csi-resize sidecar and see if that works.

If so would be open for adding a helm var to enable/disable the resize... however
as per the bug which you have mentioned it seems if online resizing is supported (which it is) we should be setting `--handle-volume-inuse-error=false` otherwise the resizer is watching pods which consumes memory.
This should fix the issue for everyone without having to modify anything.

Would you be willing to raise a PR for that?

**Comment by marcinpro1** [comment]
nvme-1:
```
[root@nvme-1 ~]# blockdev --report /dev/nvme2n1
RO    RA   SSZ   BSZ   StartSec            Size   Device
rw  8192   512  4096          0   1920383410176   /dev/nvme2n1

```
nvme-2:
```
[root@nvme-2 ~]# blockdev --report /dev/nvme3n1
RO    RA   SSZ   BSZ        StartSec            Size   Device
rw  8192   512  4096               0   1920383410176   /dev/nvme3n1
```
nvme-3:
```
[root@nvme-3 ~]# blockdev --report /dev/nvme2n1
RO    RA   SSZ   BSZ        StartSec            Size   Device
rw  8192   512  4096               0   1920383410176   /dev/nvme2n1
```

**Comment by tiagolobocastro** [comment]
Fixed on v2.7.2
